libs/ui-toolkit/modals/src/lib/ui-modal/ui-simple-modal/ui-simple-modal.stories.ts

```diff
@@ -0,0 +1,173 @@
+import { Component, Inject, Input, OnChanges, OnInit } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialog, MatDialogModule, MatDialogRef } from '@angular/material/dialog';
+import { PipesModule } from '@gorilainvest/ui-toolkit/pipes';
+import { boolean, text, withKnobs, array, object } from '@storybook/addon-knobs';
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+import { timer } from 'rxjs';
+
+import { UiSimpleModalData, SocialMediaArray } from './ui-simple-modal.data';
+import { UiSimpleModalComponent } from './ui-simple-modal.component';
+import { UiModalModule } from '../ui-modal.module';
+
+@Component({
+  selector: 'gor-test',
+  template: `
+    test
+  `,
+  styles: ['./ui-simple-modal.component.scss']
+})
+export class TesteComponent implements OnChanges, OnInit {
+  @Input() public height = '400px';
+  @Input() public width = '500px';
+  @Input() public bodyText = 'Body content';
+  @Input() public titleText = 'Title';
+   /**
+   * Indicates if image should be rendered and which image.
+   *
+   * @default '/assets/img/comming-soon.svg'
+   */
+  @Input() public bodyImage = '/assets/img/comming-soon.svg';
```

Comment:
Documentation should be added to exported library components, not to test components used in storybook